server/routes/v1/upload.ts

```diff
@@ -192,15 +193,43 @@ export const postUpload = [
       ).add({
         versionId: version._id,
         userId: req.user?._id,
-        binary: normalizeMulterFile(files.binary[0]),
-        code: normalizeMulterFile(files.code[0]),
+        binary: normalizeMulterFile(files.binary[0], 'binary', version),
+        code: normalizeMulterFile(files.code[0], 'code', version),
       })
+
       req.log.info({ code: 'created_upload_job', jobId }, 'Successfully created job in upload queue')
 
       // then return reference to user
       res.json({
         uuid: model.uuid,
       })
+
+      try {
+        const binaryLocation = `model/${model._id}/version/${version._id}/raw/binary/${files.binary[0].path}`
+        const client = new Minio.Client(config.get('minio'))
```

Comment:
There's a `getClient()` function to get a Minio instance.  It would be good turn this into a `copyFile(from, to)` function.

Reply:
Done